application/F3DStarter.cxx

```diff
@@ -493,6 +491,7 @@ void F3DStarter::LoadFile(int index, bool relativeIndex)
     loader.loadGeometry("", true);
   }
 
+  this->Internals->Engine->getOptions().set("ui.dropzone", !this->Internals->LoadedFile);
```

Comment:
If I understand correctly, that means that a libf3d user can load a file and draw the dropzone at the same time. Is it making sense?

Reply:
yes it makes sense to let libf3d users do that, if not, we would put application logic in the libf3d.